b9/src/ExecutionContext.cpp

```diff
@@ -251,96 +245,146 @@ void ExecutionContext::doPopIntoParam(StackElement *params, Immediate offset) {
 void ExecutionContext::doIntAdd() {
   auto right = stack_.pop().getInt48();
   auto left = stack_.pop().getInt48();
-  push({Om::AS_INT48, left + right});
+  doIntPushConstant(left + right);
```

Comment:
I don't think this change is the right way to go. As I see it, `doIntPushConstant` is a bytecode handler, not a general purpose helper. It's confusing to push the result as a "constant". I would rather have these lines unchanged. Eventually, I think we will change the way doIntPushConstant works, to support the full 48bit range of integers in b9 (which may involve a constant pool lookup). Sorry for asking for a change this late in the PR.